content/docs/sidebar.json

```diff
@@ -525,5 +525,33 @@
         "slug": "dvclive-with-dvc"
       }
     ]
+  },
+  {
+    "label": "Studio",
+    "slug": "studio",
+    "source": "studio/index.md",
+    "icon": "studio",
+    "children": [
+      {
+        "label": "Get started",
+        "slug": "get-started"
+      },
+      {
+        "label": "Create a view",
+        "slug": "create-view"
+      },
+      {
+        "label": "Visualize experiments",
+        "slug": "visualize-exp"
```

Comment:
Can we rename this to the full `visualize-experiments`, or at least `visualize-exps`?